include/global.tv.h

```diff
@@ -81,7 +81,7 @@ typedef union // size = 0x24
         /*0x01*/ bool8 active;
         /*0x02*/ u16 species;
         /*0x04*/ u8 pokemonName[POKEMON_NAME_LENGTH + 1];
-        /*0x0F*/ u8 trainerName[11];
+        /*0x0F*/ u8 trainerName[TRAINER_NAME_LENGTH + 1];
```

Comment:
Despite the name, this holds a player name not an NPC trainer name.
<https://github.com/pret/pokeemerald/blob/57222323c1cadf35cf8d408c84bf6a50b7347dcf/src/tv.c#L1555>

Reply:
This variable needs to hold a size that is at least the same of `TRAINER_NAME_LENGTH` though.
Should I do `PLAYER_NAME_LENGTH + 4` or something, or should I leave it as 11? 🤔 ...

Reply:
You could leave it as `+ 4` with a comment that it's larger than it needs to be, or you could split the 3 unused bytes into their own array like this
```c
        /*0x0F*/ u8 trainerName[PLAYER_NAME_LENGTH + 1];
        /*0x17*/ u8 unused[3];
```

Reply:
Alrighty, I went for the latter since it made more sense. Didn't really remember it was an option.

Should I rebase and force push to pack things nicely into a single commit, or should I just leave things as is? :eyes:

Reply:
Up to you. GitHub also has the option to squash it into a single commit on merge if you'd like me to do that.

Reply:
I don't mind either way myself, so I guess I'll leave it up to you; merging as is or squashing on merge :eyes: